okcli/completion_cache.py

```diff
@@ -0,0 +1,18 @@
+import hashlib
+from os import path
+
+def completion_filename(user, host):
+    return hashlib.md5(("%s%s" % (user, host)).encode('utf-8')).hexdigest()
+
+def completion_cache_dir(config):
+    return path.join(path.dirname(config.filename), '.okcli_cache')
```

Comment:
`os.path.dirname` will only work if `config.filename` a full path - is it? (I'd have to check).